tests/Rules/Fixture/laravel-view-function.php

```diff
@@ -47,3 +47,25 @@
 ];
 
 view('foo', $fooBar);
+
+class MyDto {
+    public string $foo;
+
+    public function __construct(
+        string $foo
+    ) {
+        $this->foo = $foo;
+    }
+}
+
+$dataDto = new MyDto('bar');
+
+view('foo', $dataDto);
+
+$dataAnonymousClass = new class extends MyDto {
+    public function __construct() {
+        parent::__construct('bar');
+    }
+}
```

Comment:
You forgot to terminate the initialization.

```suggestion
};
```